core/jobs/src/main/java/com/cognifide/aet/job/common/datafilters/jserrorsfilter/JsErrorsFilter.java

```diff
@@ -17,50 +17,40 @@
  */
 package com.cognifide.aet.job.common.datafilters.jserrorsfilter;
 
-import com.google.common.base.Predicate;
-import com.google.common.collect.FluentIterable;
-
 import com.cognifide.aet.job.api.collector.JsErrorLog;
 import com.cognifide.aet.job.api.datafilter.AbstractDataModifierJob;
 import com.cognifide.aet.job.api.exceptions.ParametersException;
 import com.cognifide.aet.job.api.exceptions.ProcessingException;
-
-import org.apache.commons.lang3.StringUtils;
-
+import com.cognifide.aet.job.common.utils.ParamsHelper;
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
 import java.util.Map;
 import java.util.Set;
-
+import java.util.regex.Pattern;
 import javax.annotation.Nullable;
 
 public class JsErrorsFilter extends AbstractDataModifierJob<Set<JsErrorLog>> {
 
   public static final String NAME = "js-errors-filter";
 
-  private static final String PARAM_ERROR = "error";
+  private static final String PARAM_ERROR = "errorPattern";
```

Comment:
I'm not sure this is good idea to change params names such drastically.
IMO we should give users at least one release cycle to adjust otherwise the old suites won't work with released version.
We definitely should mention such changes in some kind `migration log`.

Just a thought - maybe we should provide some kind of service that will automatically adjust old suites to new form?